owsi-core/owsi-core-components/owsi-core-component-jpa/src/main/java/fr/openwide/core/jpa/batch/processor/ThreadedProcessor.java

```diff
@@ -217,10 +218,10 @@ public void log(boolean force) {
 				if (StringUtils.hasText(loggerContext)) {
 					sb.append(loggerContext).append(" - ");
 				}
-				sb.append("Avancement {} / {} ({} ignorés, {} items / s. depuis début, {} items / s. depuis dernier log)");
+				sb.append("In progress: {} / {} ({} ignored, {} items/s since start, {} items/s since last log)");
 				
 				if (propertyService.get(MIGRATION_LOGGING_MEMORY)) {
```

Comment:
Instead of 2 loggers, we could just put it at the DEBUG log level. There is no other DEBUG for this logger.